python/rascaline/rascaline/utils/power_spectrum/calculator.py

```diff
@@ -217,7 +217,8 @@ def compute(
             ell = key[0]
             species_center = key[1]
 
-            factor = 1 / sqrt(2 * ell + 1)
+            factor = (-1) ** ell / sqrt(2 * ell + 1)
```

Comment:
I would add the comment from the rust code here as well:

```suggestion
             # For consistency with a full Clebsch-Gordan product we need to add
             # a `-1^l / sqrt(2 l + 1)` factor to the power spectrum invariants
            factor = (-1) ** ell / sqrt(2 * ell + 1)
```